RaccoonPG.py

```
@@ -0,0 +1,389 @@
+        self.LVL = 1
+
+        self.set_fast_xp(EXPcap, LVL)
+
+    def set_fast_xp(self, EXPcap, LVL):
+        self.EXPcap += (((4 / 5) * (self.LVL ** 4)) + ((self.LVL ** 3) * 5) + (100 * self.LVL))
+
+    def reduce_health(self, enemy):
+        damage = Animal.damage_formula(self, enemy, self.POWER, self.LVL, self.ATTK, self.CRIT)
+        self.HP -= damage
+        if self.HP <= 0:
+            self.inCombat == False
```

Ruff (0.14.0)

Avoid equality comparisons to `False`; use `not self.inCombat:` for false checks

Replace with `not self.inCombat`

(E712)